packages/commonwealth/client/scripts/views/pages/new_snapshot_proposal/helpers.ts

```diff
@@ -64,23 +58,7 @@ const newThread = async (
     metadata: JSON.stringify({}),
   };
 
-  try {
-    await createProposal(author.address, proposalPayload);
-    await app.user.notifications.refresh();
-    await app.snapshot.refreshProposals();
-  } catch (e) {
-    console.log(e);
-    throw new Error(e.error);
-  }
-};
-
-export const newLink = async (
-  form: ThreadForm,
-  quillEditorState: QuillEditor,
-  author: Account,
-  space: SnapshotSpace
-) => {
-  const errors = await newThread(form, quillEditorState, author, space);
-  console.log('the rro', errors);
-  return errors;
+  await createProposal(author.address, proposalPayload);
```

Comment:
should we really get rid of try catch here?

Reply:
Yes, the original code didn't properly pass the error and it showed up as blank. New code shows the error message properly. Though it's possible that one of the functions passes a non-standard error object instead of `Error`.